src/Tracing/SamplingContext.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Sentry\Tracing;
+
+class SamplingContext
+{
+    /**
+     * @var TransactionContext|null TransactionContext
+     */
+    private $transactionContext;
+
+    /**
+     * Returns the default instance of for the SamplingContext
+     *
+     * @param TransactionContext $transactionContext
```

Comment:
There is no much value in having this annotation without a description of the parameter. Please either add it or drop the line altogether as the parameter is self-explanatory